tests/Aspire.Components.Common.Tests/ConformanceTests.cs

```diff
@@ -367,6 +370,28 @@ key is null
                 : host.Services.GetRequiredKeyedService<TService>(key));
     }
 
+    [ConditionalFact]
+    public void FavorsNamedConfigurationOverTopLevelConfigurationWhenBothProvided()
+    {
+        SkipIfNamedConfigNotSupported();
+
+        var key = "target-service";
+        var builder = Host.CreateEmptyApplicationBuilder(null);
+
+        builder.Configuration.AddInMemoryCollection([
+            new KeyValuePair<string, string?>($"{ConfigurationSectionName}:DisableTracing", "false"),
+            new KeyValuePair<string, string?>($"{ConfigurationSectionName}:{key}:ConnectionString", "unused"),
```

Comment:
Is this line necessary? It is a bit surprising.

Reply:
Doesn't look like this is required. Just a copy-pasta artifact 🤠